src/Component.js

```diff
@@ -6,6 +6,7 @@ import copy from 'copy-to-clipboard';
 export class CopyToClipboard extends React.PureComponent {
   static propTypes = {
     text: PropTypes.string.isRequired,
+    onClickCapture: PropTypes.func,
```

Comment:
`onClickCapture` is sometimes created as a synthetic event. I am not sure WHY it's created automatically in the case of preact, since the support isn't there. But, because this was passed through, the copy action wasn't working at all. Synthetic events are not very commonly used so I suspect this is not going to cause much hassle for people.